sts_ocm_user_trust_policy.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": [
+          "arn:aws:iam::644306948063:role/RH-Managed-OpenShift-Installer"
+        ]
+      },
+      "Action": [
+        "sts:AssumeRole"
+      ],
+      "Condition": {"StringEquals": {"sts:ExternalId": "1WDBU5E101KnGowHfbeANgKzwA6"}}
+    }
+  ]
+}
```

Comment:
should this file be added to github?

Reply:
worth adding those files to a global gitignore 

```
openshift_cloud_credential_operator_cloud_credential_operator_iam_ro_creds_policy.json
openshift_cluster_csi_drivers_ebs_cloud_credentials_policy.json
openshift_image_registry_installer_cloud_credentials_policy.json
openshift_ingress_operator_cloud_credentials_policy.json
openshift_machine_api_aws_cloud_credentials_policy.json
sts_installer_permission_policy.json
sts_installer_trust_policy.json
sts_instance_controlplane_permission_policy.json
sts_instance_controlplane_trust_policy.json
sts_instance_worker_permission_policy.json
sts_instance_worker_trust_policy.json
sts_support_permission_policy.json
sts_support_trust_policy.json
sts_ocm_user_trust_policy.json
```
I have pushed those files too many times to MR's :see_no_evil: